frontend/src/components/ViewApplicationSeasons/SeasonLink.tsx

```diff
@@ -0,0 +1,40 @@
+import moment, { Moment } from 'moment';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel';
+import { Element } from 'nav-frontend-typografi';
+import React, { FunctionComponent } from 'react';
+import { NavLink } from 'react-router-dom';
+import { timeFormatForLink } from '../../utils/timeFormatter';
+import { IApplicationSeason } from '../ApplicationSeason/reducer';
+import { ROUTE_TO } from './constants';
+import { _SEASON_ENDS, _SEASOND_ENDED } from './strings';
+
+interface IProps {
+  season: IApplicationSeason;
+}
+
+const SeasonLink: FunctionComponent<IProps> = (props) => {
+  const { season } = props;
+  const currentTime = moment();
+  const seasonOnGoing = season.applicationPeriodEnd < currentTime
+  ? _SEASOND_ENDED
+  : _SEASON_ENDS(season.applicationPeriodEnd.format(timeFormatForLink)) ;
+  const link = (panelProps: any) => (
+    <NavLink
+      to={{ pathname: panelProps.href, room: { ...season } }}
+      className="lenkepanel lenkepanel--border"
+    >
+      {panelProps.children}
+    </NavLink>);
+  return (
+    <LenkepanelBase
+      key={season.id}
+      linkCreator={link}
+      href={`${ROUTE_TO}${season.id}`}
```

Comment:
This will 404, no route to update-season

Reply:
@Paalar Why did you resolve this? Please add a comment to why.

Reply:
Forgot to mention. It is resolved in #255